sierra2mlir/src/statements/inline_jumps.rs

```diff
@@ -306,19 +296,16 @@ impl<'ctx> Compiler<'ctx> {
                 let mut args_to_target_block = vec![];
                 for var_idx in target_block_info.variables_at_start.keys() {
                     if *var_idx == invocation.branches[variant_index].results[0].id {
-                        let variant_enum_type = self
-                            .llvm_struct_type(&[self.u16_type(), variant_type.get_type()], false);
-
-                        let variant_ptr_op =
-                            self.op_llvm_gep(&variant_block, &[0, 1], enum_ptr, variant_enum_type)?;
```

Comment:
Because we're using packed structs this is guaranteed to be the same behaviour. If we ever switch to non-packed structs though we will have to be careful about this